test/sequential/rolldown.updateL1FromL1.test.ts

```diff
@@ -456,3 +519,135 @@ describe.skip("updateL1FromL1 - errors", () => {
     );
   });
 });
+
+describe("updateL1FromL1 - cancelResolution and deposit errors", () => {
+  let chain: any;
+  let api: ApiPromise;
+  let sequencer: User;
+  let txIndex: number;
+
+  beforeAll(async () => {
+    await initApi();
+    setupUsers();
+    await setupApi();
+    api = getApi();
+  });
+
+  beforeEach(async () => {
+    //TODO: Replace this by some monitoring of the active queue.
+    await waitForNBlocks((await Rolldown.disputePeriodLength()).toNumber());
+    await SequencerStaking.removeAddedSequencers(10);
+    chain = "Ethereum";
+    sequencer = await setupASequencer(chain);
+    txIndex = await Rolldown.lastProcessedRequestOnL2(chain);
+  });
+
+  it("When a cancel resolution fail, maintenance mode will be triggered automatically", async () => {
+    await checkAndSwitchMnModeOff();
+    await Rolldown.waitForReadRights(sequencer.keyRingPair.address, 50, chain);
```

Comment:
we must create a constant for those waiting times ( 50 ) and the 40 below, the day we have some changes on the dp, sequencer selection we  will cry,

Reply:
Fixed here, and I can create a fast PR where I'll change it for all our tests